packages/arb-token-bridge-ui/src/components/TransferPanel/AdvancedSettings.tsx

```diff
@@ -165,7 +169,7 @@ export const AdvancedSettings = () => {
     })
   }, [debouncedDestinationAddress, setQueryParams])
 
-  if (!isEOA && !isSmartContractWallet) {
+  if (!isEOA && !isSmartContractWallet && !isLoadingAccountType) {
```

Comment:
we need to check for `isLoadingAccountType` here otherwise after switching destination chain, the destination address box won't show up

<img width="650" alt="image" src="https://github.com/user-attachments/assets/3e520169-e1d2-49b3-ac00-bc98a8426295" />